docs/src/formula.md

```diff
@@ -0,0 +1,76 @@
+```@meta
+CurrentModule = StatsModels
+DocTestSetup = quote
+    using StatsModels
+end
+```
+
+# Modeling tabular data
+
+Most statistical models require that data be represented as a Matrix-like
+collection of a single numeric type.  Much of the data we want to model,
+however, is __tabular data__, where data is represented as a collection of
```

Comment:
Being ultra pedantic here, but "data" is actually plural. So that should be "where data are" rather than "where data is."